Intro update (#2)
chhoumann/P6: sections/introduction.tex

\section{Introduction}
\label{sec:intro}
A cyber-physical system (CPS) is a computerized system where real world physical events or mechanisms are monitored or controlled through the application of computer algorithms.
In a CPS, physical and software components work together in different spatial and temporal scales.

> An example of such a system is the way large weather sensor networks works in correlation to produce both local weather time series as well as affecting how the data from other sensors are interpreted. This in turn produces multiple weather time series that are all correlated. Being able to accurately forecast the weather have huge consequences most other CPSs as well as most peoples day to day lives. Having an accurate forecasting model is necessary in order to be able to accurately identify trends and outliers as well as predicting the future behaviour of the weather. All of this is important both in the day to day running of other CPS as well as predicting the future possible climate changes coming our way. 

Reviewer: ```suggestion
An example of such a system is how large weather sensor networks work in correlation. These produce both local weather time series, but they also affect how data from other sensors are interpreted. This, in turn, produces multiple weather time series that are all correlated. Being able to accurately forecast the weather has a large impact on most other CPSs as well as most people's daily lives. Having an accurate forecasting model is necessary in order to be able to accurately identify trends and outliers as well as predicting the future behavior of the weather. Ultimately all of this is useful in both the day to day running of other CPSs, and also in predicting future possible climate changes. 
```

> In order to be able to generate a reliable time series weather forecasting model we need a model that is able to account for the seasonality aspect the weather as well as the ever changing attributes that have an influence on the future. This means that in order to generate an accurate model one most be able to take into account historical states when making predictions for the future. 

Reviewer: ```suggestion
To generate a reliable time series weather forecasting model, we need a model that is able to account for the seasonality aspect of the weather and the variable attributes that affect the future. 
This means that, in order to generate an accurate model, one must be able to take into account previous historical states when making predictions for the future. 
```

> Today the state of the art methods in time series forecasting are methods such as autoregressive integrated moving average(ARIMA), Recurrent neural networks(RNN's), long short term memory(LSTM's) and gated neural networks(GRU's). Currently in the field of natural language processing (NLP) the transformer model[cite] is making large strides in improving the state of the art compared to the formerly used methods. Due to this we aim to measure the performance of a transformer modified to do univariate time series weather forecasting against the currently and formerly used methods. 

Reviewer: ```suggestion
Today, the state of the art methods in time series forecasting include methods such as autoregressive integrated moving average (ARIMA), recurrent neural networks (RNNs), long short term memory (LSTM) and gated neural networks
(GRUs).
However, more recently in the field of natural language processing (NLP), the transformer model[cite] has shown great potential and is able outperform the aforementioned methods.
Because of this, we wish to compare the performance of the previously used methods to the more recent transformer model when performing univariate time series weather forecasting. Our goal is to determine whether this model is able to also outperform the previously used methods in time series forecasting.
```
